http/mixed-content.json

```diff
@@ -0,0 +1,322 @@
+{
+  "http": {
+    "mixed_content": {
+      "__compat": {
+        "description": "Blocks some or all insecure mixed content",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Security/Mixed_content",
+        "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
```

Comment:
Note, I had to use `https://www.w3.org/TR/mixed-content/#intro` rather than `https://www.w3.org/TR/mixed-content/` as bare URL was blocked. All the spec urls point to this same place.